queue/external_functions.h

```diff
@@ -2,16 +2,6 @@
 #define __QUEUE_EXTERNAL_FUNCTIONS_H__
 
 
-/*
- * Destroy a queue_item_t object.
- *
- * This will destroy a queue_item_t object, that is not needed anymore. This
- * includes freeing up any memory associated with the queue_item_t object for
- * example.
- */
-void
-queue_item_destroy(queue_item_t **);
-
 /*
  * Create a new queue_t object.
```

Comment:
In general all of the doc comments for these functions could be revised, for example mention that the handle returned by `queue_create()` can be used with the other `queue_*()` functions.

Reply:
Done.